docs/cql/ddl.rst

```diff
@@ -897,6 +912,46 @@ The following modes are available:
    * - ``immediate``
      - Tombstone GC is immediately performed. There is no wait time or repair requirement. This mode is useful for a table that uses the TWCS compaction strategy with no user deletes. After data is expired after TTL, ScyllaDB can perform compaction to drop the expired data immediately.
 
+.. _cql-hints-options:
+
+Table hints
+###########
+
+The per-table ``hints`` options can be used to tune the tablet allocation logic for the table.
+
+=============================== =============== ===================================================================================
+ Option                          Default         Description
+=============================== =============== ===================================================================================
+ ``expected_data_size_in_gb``   0               The expected table data size, in combination with the ``target_tablet_size_in_bytes``
+                                                config option, the replication factor configuration, and the cluster topology
+                                                determine the minimum number of tablets to allocate for the table.
+                                                It can be set when the table is created to allocate more tablets for it,
```

Comment:
you meant the number of tablet replicas, since number of tablets isn't influenced by replication factor.

Reply:
on a second thought, the description should focus that this controls the number of tablets, not tablet replicas, since the size hint is not the total replicated size. so I'd leave topology out of the description.